sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/_additional_includes.py

```diff
@@ -192,7 +187,13 @@ def resolve(self) -> None:
                 self._resolve_folder_to_compress(additional_include, Path(tmp_folder_path))
             else:
                 dst_path = (tmp_folder_path / src_path.name).resolve()
-                self._copy(src_path, dst_path, ignore_file=IgnoreFile() if self._is_artifact_includes else None)
+                self._copy(
+                    src_path,
+                    dst_path,
+                    # note that artifact additional includes file will skip ignore logic
+                    # TODO: check whether we should keep all files from artifact
```

Comment:
@lalala123123 @wangchao1230 Could you please help confirm logic here? Is there any corresponding document?

Reply:
suppose `x.py` is both in artifact, local folder and ignore file, what should we do with it?